openscap_daemon/dbus_daemon.py

```diff
@@ -363,7 +363,7 @@ def containers(self):
     @staticmethod
     def _parse_only_cache(config, onlycache):
         if onlycache == 2:
-            return not config.fetch_cve
+            return config.fetch_cve
```

Comment:
It is a leftover variable name from the first implementation of onlycache, where we overrode it only if we didn't want it.  In this context, it is probably a poor choice of a variable name.

Later, when prep'ing to scan, there is:
        if self.ac.onlycache:
            cve_get.fetch_dist_data()

So onlycache and fetch_cve are really synonymous.  If you want me to refactor the 'onlycache' variable to make more sense, I can do that.